solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -271,6 +275,7 @@ contract RandomBeacon is Ownable {
         groupLifetime = 2 weeks;
         dkgResultSubmissionReward = 0;
         sortitionPoolUnlockingReward = 0;
+        ineligibleOperatorNotifierReward = 0;
```

Comment:
Let's add a TODO to revisit the initial value at deployment stage.

Reply:
We already have a TODO:
```
        // TODO: revisit all initial values
        callbackGasLimit = 200e3;
        groupCreationFrequency = 10;
        groupLifetime = 2 weeks;
        dkgResultSubmissionReward = 0;
        sortitionPoolUnlockingReward = 0;
        ineligibleOperatorNotifierReward = 0;
       ...        
```

Reply:
Ok fine!